client/my-sites/hosting/staging-site-card/staging-site-production-card.tsx

```diff
@@ -149,14 +172,19 @@ function StagingSiteProductionCard( { disabled, siteId, translate }: CardProps )
 		);
 	}
 	return (
-		<Card className="staging-site-card">
-			{
-				// eslint-disable-next-line wpcalypso/jsx-gridicon-size
-				<Gridicon icon="science" size={ 32 } />
-			}
-			<CardHeading id="staging-site">{ __( 'Staging site' ) }</CardHeading>
-			{ cardContent }
-		</Card>
+		<>
+			<SVGContainer>
```

Comment:
Would it be more correct to make this component a part of the component it decorates? Also, if it's just decoration, would it be better to have it as a CSS background and not an SVG element placed within the DOM?

Besides that, we should adjust the component naming to follow what the component does e.g. `Divider` and `DividerPattern`.

Reply:
> Would it be more correct to make this component a part of the component it decorates? 
I did that for practical reasons. For example, we may need to reuse that SVG, but without the need for margin etc as this container provides.
> Besides that, we should adjust the component naming to follow what the component does e.g. Divider and DividerPattern.

Agree! Will do that! Thanks!

Reply:
> I did that for practical reasons. For example, we may need to reuse that SVG, but without the need for margin etc as this container provides.

I think it should be ok to add it to `client/assets/`.